### ISSUE TITLE
Minor Fixes

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4103]
**Is your feature request related to a problem? Please describe.**
VIVO should be aligned with AA level of WCAG. Different aspects of accessibility has been broken into subtasks of #4096. The PR which should address this sub-task should resolve all WCAG issues which are not in resolved in other subtasks.
